packages/ui/src/components/KafkaInstanceDrawer/KafkaInstanceDrawer.tsx

```diff
@@ -166,7 +166,9 @@ export const KafkaInstanceDrawerPanel: VoidFunctionComponent<
                 externalServer={getExternalServer(instance.bootstrapUrl)}
                 tokenEndPointUrl={tokenEndpointUrl}
                 linkToServiceAccount={"service-accounts"}
-                linkToAccessTab={`TODO/acls`}
+                linkToAccessTab={
+                  isKafkaPending ? `#` : `${instance.id}/details/acls`
```

Comment:
No, better have this support `undefined` and hide the link until the instance isn't ready. Else we show a link that doesn't work.